app/src/main/java/org/oppia/android/app/walkthrough/topiclist/WalkthroughTopicListFragmentPresenter.kt

```diff
@@ -63,6 +65,19 @@ class WalkthroughTopicListFragmentPresenter @Inject constructor(
     binding.walkthroughTopicRecyclerView.apply {
       layoutManager = walkthroughLayoutManager
       adapter = createRecyclerViewAdapter()
+      addOnScrollListener(object : RecyclerView.OnScrollListener() {
+        override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+          super.onScrolled(recyclerView, dx, dy)
+          val pos = walkthroughLayoutManager.findFirstVisibleItemPosition()
```

Comment:
can we use be more understanding variable name than `pos` ?

Reply:
Oh, I forgot to give it a good name, I will fix it for sure